engine/model_webhook.go

```diff
@@ -127,45 +164,19 @@ func (wr *WebhooksRepository) getByURL(ctx context.Context, url string) (*Webhoo
 	return webhook, nil
 }
 
-// RemoveWebhook removes a webhook from the database
-func (wr *WebhooksRepository) RemoveWebhook(ctx context.Context, url string) error {
-	webhook, err := wr.getByURL(ctx, url)
-	if err != nil {
-		return err
-	}
-
-	webhook.DeletedAt.Valid = true
-	webhook.DeletedAt.Time = time.Now()
-
-	return Save(ctx, webhook)
-}
```

Comment:
### Minor - just worth consideration

```go
// Delete removes a webhook from the database
func (wr *WebhooksRepository) Delete(ctx context.Context, model notifications.ModelWebhook) error {
	webhook, ok := model.(*Webhook)
	if !ok {
		return errors.New("Unknown implementation of notifications.ModelWebhook")
	}
	webhook.delete()
	return Save(ctx, webhook)
}
```

#### Reason:
- this way the repository will choose if deletion in DB is of or hard deletion
- we can make a method MarkDeleted private

Reply:
Good point - working on this :)